### PR TITLE
Bump dataflow-java dependency.

### PR DESCRIPTION
**Pull request by deflaux**
Fixes item (6) of https://github.com/googlegenomics/readthedocs/issues/128

### CODE REVIEW

**Comment by deflaux** [comment]
This should be merged until the new version of dataflow-java is on Maven central.  Travis CI will be green when that is available.

File: pom.xml [hunk 1]
```diff
@@ -10,8 +10,7 @@
   <properties>
     <compileSource>1.8</compileSource>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <dataflow.version>1.0.0</dataflow.version>
-    <genomics.dataflow.version>v1beta2-0.18</genomics.dataflow.version>
+    <genomics.dataflow.version>v1-0.1</genomics.dataflow.version>
```
Comment (lesv) [inline pom.xml]: This doesn't look like a valid maven version number.

File: pom.xml [hunk 2]
```diff
@@ -10,8 +10,7 @@
...
     <maven.compiler.version>3.3</maven.compiler.version>
     <bigtable.version>0.2.3</bigtable.version>
```
Comment (lesv) [inline pom.xml]: You may wish to bump this to 0.3.0

File: pom.xml (hunk 1 above)
Comment (deflaux) [inline pom.xml]: We just upgraded our googlegenomics/utils-java and googlegenomics/dataflow-java to v1 of the Genomics API.  The release is on Maven Central now.

File: pom.xml (hunk 2 above)
Comment (deflaux) [inline pom.xml]: Done, thanks!

File: pom.xml (hunk 1 above)
Comment (lesv) [inline pom.xml]: Yep - verified - still a weird version #.

**Comment by lesv** [comment]
LGTM - if you take those from me for this repo.

**Comment by cmclean** [comment]
LGTM

**Pull request merged**